modules/local/plot_entity_binding_ratios.nf

```diff
@@ -18,15 +18,17 @@ process PLOT_ENTITY_BINDING_RATIOS {
     path "versions.yml",                    emit:   versions
 
     script:
+    def handle_pvalue = params.hide_pvalue ? "-d TRUE" : "-d FALSE"
```

Comment:
I would make the names matching and only return "TRUE"/"FALSE" and make the -d constant
```suggestion
    def hide_pvalue = params.hide_pvalue ? "TRUE" : "FALSE"
```